autoware_map_msgs/srv/LoadDifferentialPointCloudMap.srv

```diff
@@ -0,0 +1,13 @@
+# Area query for map loading
+# (PCD grids that overlaps with the area is going to be loaded)
+AreaInfo area
+
+# The IDs of PCD maps that the node already has
+string[] already_loaded_ids
```

Comment:
```suggestion
string[] cached_ids
```
Also following from https://github.com/autowarefoundation/autoware_msgs/pull/39#discussion_r981952446

I think `cached_ids` reflects it better.

Reply:
resolved https://github.com/autowarefoundation/autoware_msgs/pull/39/commits/27a8ea6714aeac6be4f1b394e994aad73fdfa167